app/views/hyrax/contact_form/new.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="alert alert-info">
+  <%= render 'directions' %>
+</div>
+
+<h1>
+    <%= t('hyrax.contact_form.header') %>
+</h1>
+
+<% if user_signed_in? %>
```

Comment:
Variables set here aren't very clear.. can you just move them to where they are used directly 

`value: <%= user_signed_in? ? current_user.name : '' %>`
`value: <%= user_signed_in? ? current_user.email : '' %>`

Reply:
for sure. This is just how the code was when I copied it over from hyrax.